### PR TITLE
#152 Fix specyfication text spaces

### LINKED ISSUE #152: Bug: Major: The text is overlapped in standard features subsection of specs block
Steps:
1. In mobile viewport, Navigate to https://116-specs--vg-macktrucks-com--hlxsites.hlx.page/drafts/andrei.balanescu/Trucks/Anthem/Specs/ 
2. Validate the standard features subsection of specs block

Expected vs Actual:
The text is overlapped in standard features subsection of specs block

image

image

### PR DESCRIPTION
**Pull request by TomaszDziezykNetcentric**
Fix #152

Test URLs:
- Before: https://main--vg-macktrucks-com--hlxsites.hlx.page/drafts/andrei.balanescu/Trucks/Anthem/Specs/
- After: https://152-spec-block-text--vg-macktrucks-com--hlxsites.hlx.page/drafts/andrei.balanescu/Trucks/Anthem/Specs/

### CODE REVIEW

File: blocks/specifications/specifications.css [hunk 1]
```diff
@@ -41,7 +41,7 @@ main .section.specifications-container {
 .specs-list-multiple .feature-list .p-element {
   width: calc(50% - 65px);
   word-wrap: break-word;
-  height: 65px;
+  min-height: 65px;
```
Comment (jonatan-lledo-netcentric) [inline blocks/specifications/specifications.css:44]: your solution is good, but adding the following is better:
```flex: 1 1 auto;```

**Review by jonatan-lledo-netcentric — CHANGES_REQUESTED** [review]

File: blocks/specifications/specifications.css (hunk 1 above)
Comment (TomaszDziezykNetcentric) [reply blocks/specifications/specifications.css:44]: I think that the `flex: 1 1 auto` is not what was expected.
The prod: 
obraz
And the flex version:
obraz
The current fix
obraz

Comment (TomaszDziezykNetcentric) [reply blocks/specifications/specifications.css:44]: @SantiagoHomps-NC maybe you can tell us if it was intentional :)?

**Review by jonatan-lledo-netcentric — APPROVED** [review]

File: blocks/specifications/specifications.css (hunk 1 above)
Comment (SantiagoHomps-NC) [reply blocks/specifications/specifications.css:44]: I think this works fine, and its solves the mobile view issue.

**Review by SantiagoHomps-NC — APPROVED** [review]

**Pull request merged**